pandas/core/resample.py

```diff
@@ -1582,7 +1602,15 @@ def _take_new_index(obj, indexer, new_index, axis=0):
         raise ValueError("'obj' should be either a Series or a DataFrame")
 
 
-def _get_range_edges(first, last, offset, closed='left', base=0):
+def _get_timestamp_range_edges(first, last, offset, closed='left', base=0):
+    """
+    Adjust the provided Timestamp range edge values to the appropriate edge
+    values for the given offset parameters.
```

Comment:
can you add a Parameters / Returns section here as well (and on _get_period_range_edges)

Reply:
done